wzdx/standard_to_enhanced/navjoy_translator.py

```diff
@@ -17,31 +16,39 @@ def main():
         navjoy_obj = json.loads(open(inputfile).read())
     except ValueError as e:
         raise ValueError(
-            'Invalid file type. Please specify a valid Json file!') from None
+            "Invalid file type. Please specify a valid Json file!"
+        ) from None
     wzdx_obj = wzdx_creator(navjoy_obj)
 
-    location_schema = 'wzdx/sample_files/validation_schema/wzdx_v3.1_feed.json'
+    location_schema = "wzdx/sample_files/validation_schema/work_zone_feed_v42.json"
     wzdx_schema = json.loads(open(location_schema).read())
 
     if not wzdx_translator.validate_wzdx(wzdx_obj, wzdx_schema):
         logging.error(
-            'validation error more message are printed above. output file is not created because the message failed validation.')
+            "validation error more message are printed above. output file is not created because the message failed validation."
+        )
         return
-    with open(outputfile, 'w') as fwzdx:
-        fwzdx.write(json.dumps(wzdx_obj, indent=2))
+    with open(outputfile, "w") as fWzdx:
+        fWzdx.write(json.dumps(wzdx_obj, indent=2))
         print(
```

Comment:
Print to logging swap

Reply:
Updated other logging statements to use print